components/dropdown/dropdown-content-mixin.js

```diff
@@ -703,10 +790,22 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 					</div>
 					<div class=${classMap(bottomClasses)} style=${styleMap(contentWidthStyle)}>
 						<slot name="footer" @slotchange="${this.__handleFooterSlotChange}"></slot>
+						<d2l-button
+							style=${styleMap(closeButtonStyles)} 
+							@click=${this.close}>
+							${this.localize('components.dropdown.close')}
+						</d2l-button>
 					</div>
 				</div>
 			</div>
 		`;
+		if (this.mobileTray) return html`
+			${dropdown} 
+			<d2l-backdrop 
+				for-target="d2l-dropdown-wrapper" 
+				?shown="${this._showBackdrop}" >
+			</d2l-backdrop>`;
+		else return html`${dropdown}`;
```

Comment:
Nit: totally the same outcome, but you could use a conditional `return` here.

```suggestion
		return this.mobileTray ? html`
			${dropdown} 
			<d2l-backdrop 
				for-target="d2l-dropdown-wrapper" 
				?shown="${this._showBackdrop}" >
			</d2l-backdrop>` : html`${dropdown}`;
```